src/index.html

```diff
@@ -70,7 +70,8 @@ <h1>Admissions Process</h1>
       <p>For more details on Ada's Target Student and Admissions Process, please check out <a href="http://www.adadevelopersacademy.org/applicants" target="_blank">our main Admissions Page</a>.</p>
       <h4>Admittance Requirements</h4>
       <ul>
-        <li>Must be at least 21 years old</li>
+        <li>Must be at least 18 years old; at least 21 years
```

Comment:
"For the Cohort 13 admissions cycle, we recommend that applicants be at least 21 years of age, but we will consider applications of those who are over 18. For Cohort 14, we will only be accepting applicants who are over the age of 21."

Can we have this be the last bullet point in the list of requirements?